components/compiler-families/arm-compilers-devel/SPECS/arm-compilers-devel.spec

```diff
@@ -73,7 +82,11 @@ if [ -z "${versions}" ]; then
     exit 1
 fi
 
-%post
+%build
```

Comment:
No need to introduce an empty %build section. In @koomie's previous pull request I especially asked to have it removed. If it is empty, you can just leave it away.